IBPSA/Fluid/FixedResistances/PlugFlowPipe.mo

```diff
@@ -1,7 +1,7 @@
 within IBPSA.Fluid.FixedResistances;
 model PlugFlowPipe
   "Pipe model using spatialDistribution for temperature delay"
-  extends IBPSA.Fluid.Interfaces.PartialTwoPortVector;
```

Comment:
`IBPSA.Fluid.Interfaces.PartialTwoPortVector` may be retired into Obsolete as well as it was only used by the plug flow model.
To be validated with the IBPSA team.

Reply:
I might keep it like that since it predates the `PlugFlowPipe` model. Also, in the same package, there is also the `PartialFourPortParallel` which is not in used.

Reply:
I filed a ticket so that it can be discussed before implemented here: #1504